### ISSUE TITLE
Issues navigating Community View with Voiceover

### ISSUE THREAD

**Issue by WestonHanners** [mormaer/Mlem#352]
**Describe the bug**
unable to access the favorite button
text is duplicated across the element
each row is like 4 separate elements
community groups (home, all, local) descriptions are in 2 labels each.

**Expected Behavior**
not the above

**Mlem Information:**
0.1.0 (19)

**Comment by WestonHanners** [comment]
Fixed
